### DISCUSSION TITLE
作業三的空格

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#634] [Show and tell]
![image](https://user-images.githubusercontent.com/100193736/170197375-d3491d9e-aedf-46fe-856f-77dbc73e22b5.png)
作業三是過好多次在jupyter lab明明都沒問題，但丟上github 又一直顯示錯誤，在開始懷疑人生後去查test查程式碼才發現跟上次作業一樣忘了空格....(兩變數跟乘號都有)。希望各位別傻傻地放廢很多時間啊!

**Comment by stan890425** [comment]
我剛剛也是~想說明明就寫對

**Comment by Jay-722** [comment]
每次都跑得出來過不了

**Comment by awkward-willy** [comment]
分享一下不知道算不算作弊的方式
可以透過 hw3_0_test.py 先把答案抓下來
然後透過 Python 去比對回傳值是否相等
（畢竟 Autograding 也是這樣做的）

如：
```python
ans = [['2 * 2 = 4', '2 * 3 = ........]
print(output == ans)
```
如果可以得到 True 上傳就肯定沒問題啦
雖然這樣某種程度算是看了答案啦
但大家不要直接回傳複製的答案喔

**Comment by 11timyyy** [comment]
謝謝告知!

**Comment by d26488399** [comment]
謝謝提醒

**Comment by henrywen547** [comment]
我這邊反而出了其他的錯...
我在寫迴圈設變數時(設了一個j)
在vscode上寫結果不小心選到提示給的jacobi
結果vscode就幫我多寫了一行`from sympy import jacobi`
於是檢查時就錯了...
真的會被自己的愚蠢笑死
題外話
jacobi是幹嘛用的?

**Comment by Rosancl** [comment]
感謝提醒，上次在作業二也困擾好久才發現是空格的問題。

**Comment by Lai27330** [comment]
謝謝告知，難怪測試了那麼久，小地方也是要仔細注意。

**Comment by Yao900108** [comment]
我也是這樣! 謝謝提醒

**Comment by yenwei12138** [comment]
我有發生一個問題是，' * '跟' = '的符號我要加str()才能執行，但室友卻不用
